af/oometa/013-ooexpl-tables.install.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
+select _v.register_patch( '013-ooexpl-tables', ARRAY[ '012-sha256-input-uniq' ], NULL );
+
+CREATE TABLE ooexpl_bucket_msm_count (
+    "count" bigint,
+    "probe_asn" integer,
+    "probe_cc" character(2),
+    "bucket_date" date,
+    CONSTRAINT ooexpl_bucket_msm_count_pkey PRIMARY KEY (probe_asn, probe_cc, bucket_date)
+);
+
+comment on table ooexpl_bucket_msm_count is 'OONI Explorer stats table for counting the total number of measurements since the beginning of time by probe_cc and probe_asn';
+
+CREATE TABLE ooexpl_daily_msm_count (
+    "count" bigint,
+    "probe_cc" character(2),
+    "probe_asn" integer,
+    "test_name" ootest,
+    "test_day" timestamp without time zone,
+    "bucket_date" date,
```

Comment:
BTW, you could save some bytes here:

- `count` as a `bigint` (aka `int8`) does not make much sense, as `msm_no` is `integer` (aka `int4`). Yes, `msm_no` already takes 13% of available "address space", but 7x growth is still needed to take all the space, that'll take few years at a current rate.
-  using `date` for `test_day` saves another 4 byes.

Does it have measurable effect in terms of performance? It's unclear.